server/game/core/ability/KeywordHelpers.ts

```diff
@@ -102,9 +105,37 @@ function parseNumericKeywordValueIfEnabled(keyword: KeywordName, cardText: strin
     return Number(match.value[1]);
 }
 
+/**
+ * Checks if the Smuggle keyword is enabled and returns
+ *
+ * @returns null if the keyword is not enabled, or the numeric value if enabled
+ */
+function parseSmuggleIfEnabled(cardText: string, cardName: string): KeywordWithCostValues {
+    const regex = getRegexForKeyword(KeywordName.Smuggle);
+    const matchIter = cardText.matchAll(regex);
+
+    const match = matchIter.next();
+    if (match.done) {
+        return null;
+    }
+
+    if (matchIter.next().done !== true) {
+        throw new Error(`Expected to match at most one instance of enabled keyword ${KeywordName.Smuggle} in card ${cardName}, but found multiple`);
+    }
+
+    const smuggleCost = Number(match.value[1]);
+    const aspectString = match.value[2];
+    const smuggleAspects = EnumHelpers.checkConvertToEnum(aspectString.toLowerCase().split(' '), Aspect);
```

Comment:
I keep forgetting to ask - can you send me two examples of cards from the swu site that have mismatched smuggle text so I can report it to them, we've had a couple glitches like this come up now and it's probably worth at least bringing it up